spec/controllers/bikes_controller_spec.rb

```diff
@@ -96,7 +96,7 @@
       end
       context "passed all permitted params" do
         let(:query_params) do
-          {
+          ActionController::Parameters.new(
```

Comment:
Why are you switching this hash to permitted params? It seems weird that you're calling `permit` in the test.

Reply:
Agreed, but I'm accommodating the stub, which now expects a params object instead of a hash:

```rb
expect(Bike).to receive(:searchable_interpreted_params).with(query_params, ip: "special") { {} }
```

I think it's definitely worth refactoring though — it might shed some light on the path helper / to_hash issues.

Reply:
Ahhhh. Ok. sorry.